modules/retail/pages/containerized-saltboot.adoc

```diff
@@ -63,65 +70,45 @@ See xref:retail-terminal-names.adoc[] for explanation of possible configurations
 
 [IMPORTANT]
 ====
-Containerized retail workflow no longer manages DHCP server. External DHCP service must be used.
+Containerized retail workflow no longer manages DHCP server.
+External DHCP service must be used.
 ====
 
 Containerized workflow relies on updated image building in {productname} Server 4.3 where PXE images are no longer collected as bundle, but kernel, initrd and filesystem image are collected individually.
 
 Containerized workflow uses new TFTP container which instead of providing files present on the proxy, routes specific TFTP requests as HTTP requests through proxy local squid proxy to {productname} Server.
 
-Leveraging these two improvements and the fact that containerized proxy is not a salt client, it is no longer possible to call [systemitem]``image-sync`` state.
-After image is build images are immediately available to the saltboot clients. This may have implications on how images are deployed to production.
+Containerized proxy is not a Salt client, it is not possible to call [systemitem]``image-sync`` state.
 
-Following documents differences between containerized and regular workflow. Both are assuming proxy (containerized or in form of {smr} Branch Server) are available.
+Thanks to these improvements and not present Satl client, after image is build it is immediately available to the Saltboot clients.
```

Comment:
It isn't completely clear to me what this statement (the first half of the sentence) actually means.  Maybe, a suggestion I made caused the trouble.

Reply:
I removed first sentence and focus on what is important: build image is immediately available to the clients without `image-sync` application.